2.1.2 broken

During npm update:


Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'ts-node' imported from /home/hex/ZweloProject/ClientApp/node_modules/maplibre-gl/
  at new NodeError (node:internal/errors:371:5)
npm ERR!     at packageResolve (node:internal/modules/esm/resolve:884:9)
npm ERR!     at moduleResolve (node:internal/modules/esm/resolve:929:18)
npm ERR!     at defaultResolve (node:internal/modules/esm/resolve:1044:11)
npm ERR!     at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
npm ERR!     at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
npm ERR!     at ESMLoader.import (node:internal/modules/esm/loader:276:22)
npm ERR!     at initializeLoader (node:internal/process/esm_loader:74:49)
npm ERR!     at loadESM (node:internal/process/esm_loader:87:11)
npm ERR!     at runMainESM (node:internal/modules/run_main:47:21) {
npm ERR!   code: 'ERR_MODULE_NOT_FOUND'

I am running into a similar issue this morning, this works with version `2.1.1`, but `2.1.2` produces this error.
I tried it on two machines:
* (node version: `v16.13.2`, npm: `8.4.0`, OS: `Debian GNU/Linux 10 (buster) 5.10.76-linuxkit`, CPU: `Apple M1` (aarch64))
* (node version: `v16.13.2`, npm: `8.1.2`, OS: `Ubuntu 20.04.3 LTS (Focal Fossa))`, CPU: `Intel i7` (x86_64))

It _might_ be a different issue, since it's talking about `ts-node`.

I created a very simple package.json: (I also tried it as a dependency on both machines)
```
{
  "name": "test",
  "devDependencies": {
    "maplibre-gl": "2.1.2"
  }
}
```

This is the error I get when I run npm install
```
$ npm install                                                                                                                                             
npm ERR! code 1
npm ERR! path /projects/test/node_modules/maplibre-gl
npm ERR! command failed
npm ERR! command sh -c npm run codegen && npm run generate-query-test-fixtures
npm ERR! > maplibre-gl@2.1.2 codegen
npm ERR! > npm run generate-style-code && npm run generate-struct-arrays && npm run generate-style-spec && npm run generate-shaders
npm ERR!
npm ERR!
npm ERR! > maplibre-gl@2.1.2 generate-style-code
npm ERR! > node --loader ts-node/esm --experimental-specifier-resolution=node build/generate-style-code.ts
npm ERR! (node:1831) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
npm ERR! (Use `node --trace-warnings ...` to show where the warning was created)
npm ERR! node:internal/errors:464
npm ERR!     ErrorCaptureStackTrace(err);
npm ERR!     ^
npm ERR!
npm ERR! Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'ts-node' imported from /projects/test/node_modules/maplibre-gl/
npm ERR!     at new NodeError (node:internal/errors:371:5)
npm ERR!     at packageResolve (node:internal/modules/esm/resolve:884:9)
npm ERR!     at moduleResolve (node:internal/modules/esm/resolve:929:18)
npm ERR!     at defaultResolve (node:internal/modules/esm/resolve:1044:11)
npm ERR!     at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
npm ERR!     at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
npm ERR!     at ESMLoader.import (node:internal/modules/esm/loader:276:22)
npm ERR!     at initializeLoader (node:internal/process/esm_loader:74:49)
npm ERR!     at loadESM (node:internal/process/esm_loader:87:11)
npm ERR!     at runMainESM (node:internal/modules/run_main:47:21) {
npm ERR!   code: 'ERR_MODULE_NOT_FOUND'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jimmyrocks/.npm/_logs/2022-02-12T15_48_12_520Z-debug-0.log
```

Attempted a solution with #991, but this created a new problem:

```
➜  test npm install maplibre-gl@2.1.3
npm ERR! code 1
npm ERR! path /home/wipfli/maps/test/test/node_modules/maplibre-gl
npm ERR! command failed
npm ERR! command sh -c node ./postinstall.js
npm ERR! node:internal/modules/cjs/loader:936
npm ERR!   throw err;
npm ERR!   ^
npm ERR! 
npm ERR! Error: Cannot find module '/home/wipfli/maps/test/test/node_modules/maplibre-gl/postinstall.js'
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
npm ERR!     at node:internal/main/run_main_module:17:47 {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: []
npm ERR! }
```

Yup.
npm ERR! Error: Cannot find module '/home/hex/ZweloProject/ClientApp/node_modules/maplibre-gl/postinstall.js'
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
npm ERR!     at node:internal/main/run_main_module:17:47 {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: []
npm ERR! }

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,13 @@
 
 - *...Add new stuff here...*
 
+## 2.1.4
+
+
+### 🐞 Bug fixes
+
+- Fix missing `postinstall.js` file in npm publish. Follow-up on (#990), (#991).
+
 ## 2.1.3
 
 ### 🐞 Bug fixes

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "maplibre-gl",
   "description": "BSD licensed community fork of mapbox-gl, a WebGL interactive maps library",
-  "version": "2.1.3",
+  "version": "2.1.4",
   "main": "dist/maplibre-gl.js",
   "style": "dist/maplibre-gl.css",
   "license": "BSD-3-Clause",
@@ -176,6 +176,7 @@
   "files": [
     "build/",
     "dist/*",
-    "src/"
+    "src/",
+    "postinstall.js"
   ]
 }

